Makefile

```diff
@@ -56,6 +57,17 @@ test-with-cover:
 	$(GOTEST) $(GOTEST_OPT_WITH_COVERAGE) $(ALL_PKGS)
 	go tool cover -html=coverage.txt -o coverage.html
 
+.PHONY: addlicense
+addlicense:
+	@ADDLICENCESEOUT=`$(ADDLICENCESE) -y 2019 -c 'OpenTelemetry Authors' $(ALL_SRC) 2>&1`; \
```

Comment:
Nit: Copyright year doesn't need to be specified since it only indicates when it's added. If we were to add more files in 2020, it's fine to keep old files as "Copyright 2019".

Reply:
AFAIK, It only works for newly added files, so old files won't be affected.